plugins/lime-plugin-admin/src/adminPage.js

```diff
@@ -83,42 +82,34 @@ export const Admin = ({ adminLogin, changeConfig, showNotification, selectedHost
 	}
 
 	return (
-		<div class="container" style={{ paddingTop: '100px' }}>
+		<div className="container container-padded">
```

Comment:
replace className with class

Reply:
this issue repeats in the whole PR

Reply:
will address in a different PR, since it's not a bug, and className is present at many files in develop too.